app/assets/stylesheets/hyrax/_file_upload.scss

```diff
@@ -20,3 +20,7 @@
 }
 
 .files .name span { word-break: break-all; }
+
+.progress-bar-complete {
```

Comment:
This is in the _file_upload.scss, but this is really styling the form_progress component.  Is there a more appropriate place this could go?